sdk/storage/azure-storage-file/apireview/azure/storage/file/aio/file_client_async.pyi

```diff
@@ -0,0 +1,42 @@
+# Stubs for azure.storage.file.aio.file_client_async (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._deserialize import deserialize_file_properties, deserialize_file_stream
+from .._generated.aio import AzureFileStorage
+from .._generated.models import FileHTTPHeaders, HandleItem, StorageErrorException
+from .._generated.version import VERSION
+from .._parser import _datetime_to_str, _get_file_permission
+from .._shared.base_client_async import AsyncStorageAccountHostsMixin
+from .._shared.downloads_async import StorageStreamDownloader
+from .._shared.parser import _str
+from .._shared.policies_async import ExponentialRetry
+from .._shared.request_handlers import add_metadata_headers, get_length
+from .._shared.response_handlers import process_storage_error, return_response_headers
+from .._shared.uploads_async import FileChunkUploader, IterStreamer, upload_data_chunks
+from ..file_client import FileClient as FileClientBase
+from ..models import ContentSettings, FileProperties, NTFSAttributes, ShareProperties
+from ._polling_async import CloseHandlesAsync
+from .models import HandlesPaged
+from azure.core.async_paging import AsyncItemPaged
+from datetime import datetime
+from typing import Any, Dict, Iterable, List, Optional, Union
+
+class FileClient(AsyncStorageAccountHostsMixin, FileClientBase):
+    def __init__(self, file_url: str, share: Optional[Union[str, ShareProperties]]=..., file_path: Optional[str]=..., snapshot: Optional[Union[str, Dict[str, Any]]]=..., credential: Optional[Any]=..., loop: Any=..., **kwargs: Any) -> None: ...
+    async def create_file(self, size: int, content_settings: Optional[ContentSettings]=..., metadata: Optional[Dict[str, str]]=..., file_attributes: Union[str, NTFSAttributes]=..., file_creation_time: Union[str, datetime]=..., file_last_write_time: Union[str, datetime]=..., file_permission: Optional[str]=..., file_permission_key: Optional[str]=..., timeout: Optional[int]=..., **kwargs: Any) -> Dict[str, Any]: ...
+    async def upload_file(self, data: Any, length: Optional[int]=..., metadata: Optional[Dict[str, str]]=..., content_settings: Optional[ContentSettings]=..., validate_content: bool=..., max_connections: Optional[int]=..., file_attributes: Union[str, NTFSAttributes]=..., file_creation_time: Union[str, datetime]=..., file_last_write_time: Union[str, datetime]=..., file_permission: Optional[str]=..., file_permission_key: Optional[str]=..., encoding: str=..., timeout: Optional[int]=..., **kwargs: Any) -> Dict[str, Any]: ...
+    async def start_copy_from_url(self, source_url: str, metadata: Optional[Dict[str, str]]=..., timeout: Optional[int]=..., **kwargs: Any) -> Any: ...
```

Comment:
Return type is not helping here to review :/

Reply:
#7803